### ISSUE TITLE
Update status field for vascular plants [Plant List Survey]

### ISSUE THREAD

**Issue by Sam-Amy** [NERC-CEH/irecord-app#275]
On the iRecord website vascular plant form (https://irecord.org.uk/enter-vascular-plants) we currently have the option to record a 'Status' against each species record. Could this be implemented on the app alongside the existing vascular-plant specific fields? 

The options provided in a dropdown list should be:

not recorded
native - origin unknown
introduced - accidental
introduced - accidental - regenerating
introduced - intentional
introduced - intentional - regenerating
introduced - origin unknown
introduced - origin unknown - regenerating

**Comment by kazlauskis** [comment]
The mobile app has a Plant Survey feature that uses the same web form. It appears that the terms have been updated. Do you want us to update the Plant Survey choices to match the new terms, or is this ticket regarding the addition of these stage options to the general survey in the app?

**Comment by Sam-Amy** [comment]
Yes please to update the Plant Survey choices to match the new terms; the ones above are BSBIs current preference. Thanks

I will get back to you re: general survey in the app - for bryophytes the relevant fields come up when a species is selected, but this isn't the case for vascular plants. Would this be a big job to implement?

**Comment by Sam-Amy** [comment]
Are there any implications when you update these for data already entered in this survey under the previous terms? @kitenetter I think you said at least for similar changes on iRecord website surveys there might be? Thanks

**Comment by kitenetter** [comment]
@Sam-Amy I think it depends on how things are changing, e.g. if existing terms are being edited, or new terms are being added, or old terms are being deprecated. All those options are possible but there may be different implications depending on what is being proposed.

**Comment by Sam-Amy** [comment]
In this case, we want to add new terms and deprecate the old ones (so they are not available for data entry any more), but of course keep any previous data containing those original terms.

**Comment by sacrevert** [comment]
@kazlauskis do you have remaining capacity on the contract covering this Milestone to update this termlist?
